bin/rrarchetypes

```diff
@@ -166,6 +166,48 @@ def get_galaxies(nb,wave,seed):
     flux /= np.median(flux,axis=1)[:,None]
 
     return flux, subtype
+def get_quasars(nb,wave,seed):
+    """Get the quasar spectra
+
+    Args:
+        nb (int): Number of spectra
+        wave (float array): Rest-Frame wavelength grid in Angstrom
+    Returns:
+        flux (float array): Each spectra (nSpec,nPixel) on the same
+            wavenlength grid
+        subtype (string array): Subtype of each spectra
+
+    """
+
+    ###
+    data = {}
+
+    ###
+    data['QSO'] = {}
+    data['QSO']['NB'] = nb
+
+    tseed = seed
+    data['QSO']['FLUX'], data['QSO']['WAVE'], data['QSO']['META'] = SIMQSO(restframe=True).make_templates(nmodel=data['QSO']['NB'],
+        zrange=(0., 7.),nocolorcuts=True,seed=tseed, lyaforest=True)
+
```

Comment:
Since you're resampling below here anyway, to get the Lyman-alpha forest as well as the Baldwin effect you could just do something like:
```
SIMQSO().make_templates(nmodel=data['QSO']['NB'], zrange=(0., 7.), 
  nocolorcuts=True, seed=tseed, lyaforest=True, noresample=True)
```
and then deredshift and normalize (if necessary) the spectra yourself.

The observed-frame wavelength array of each (output) spectrum is currently hard-coded to 0.45-6 micron (at a resolution of 8000), so we would need to extend that in order for your deredshifted spectra to cover the full DESI spectral range.  For a `z=7` QSO a maximum wavelength range of 10 microns should do it, right?

Reply:
Once Travis tests are done I'll merge https://github.com/desihub/desisim/pull/383 and then, here, you can do something like:
```
SIMQSO(basewave_max=10e4).make_templates(nmodel=data['QSO']['NB'], zrange=(0., 7.), 
  nocolorcuts=True, seed=tseed, lyaforest=True, noresample=True)
```
which for the highest-redshift quasars you should be able to deredshift to a maximum wavelength of 1.25 micron, well beyond the red wavelength cutoff of the red camera.

Reply:
ok, thank you. I will try this as soon as you merged the referenced PR.